contentcuration/contentcuration/frontend/administration/mixins.js

```diff
@@ -160,7 +160,7 @@ export const tableMixin = {
         ...this.$route.query,
       };
       if (params.sortBy) {
-        params.ordering = (params.descending ? '-' : '') + params.sortBy;
+        params.ordering = (String(params.descending) === 'true' ? '-' : '') + params.sortBy;
```

Comment:
Hi @Jaspreet-singh-1032! Thanks for the changes here. Altho I have some queries, It seems like this code change would never change the ordering even when we "do apply" the filter. From the issue, it seems like the ordering must be changed only when we apply the language filter and then must be reverted back!

Reply:
@ozer550 Hi brother, in that issue I meant to say that "when we apply the language filter" and "when the language filter is removed", the ordering must be same in both the cases. So, the code that @Jaspreet-singh-1032 has written is correct. 

Confusion happens @ozer550, nothing to worry dear brother! :heart: 

@Jaspreet-singh-1032 you always figure things out! Good one! :rocket: